data/varad-kulkarni172.json

```
@@ -0,0 +1,28 @@
+{
+  "name": "Varad Kulkarni",
+  "bio": "Inquisite Full Stack Web Developer who aims to make the paperless education more reliable and easy to use for people...I am also interested in Artificial Intelligence and data analytics."
+  "tags": ["Open Source", "Community"],
+   "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/KulVarad172004",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Follow me on Instagram",
+      "url": "https://instagram.com/doofenshmirtz._172?igshid=MzNlNGNkZWQ4Mg==",
+      "icon": "FaInstagram"
+    },
+     {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/varad-kulkarni-675504264",
+      "icon": "FaLinkedIn"
```

```suggestion
      "icon": "FaLinkedin"
```

The icons are case sensitive :)